tcp_check/tests/test_tcp_check.py

```diff
@@ -179,6 +322,13 @@ def test_ipv6(aggregator, check):
             aggregator.assert_metric('network.tcp.can_connect', value=1, tags=expected_tags)
     assert nb_ipv4 == 4
     # The Windows CI machine doesn't return IPv6
-    assert nb_ipv6 == 8 or platform.system() == 'Windows' and nb_ipv6 == 0
+    # Windows or MacOS might not have IPv6 connectivity when testing locally
+    assert (
+        nb_ipv6 == 8
+        or platform.system() == 'Windows'
+        and nb_ipv6 == 0
+        or platform.system() == 'Darwin'
+        and nb_ipv6 == 0
```

Comment:
```suggestion
        nb_ipv6 == 8
        or (platform.system() == 'Windows'
        and nb_ipv6 == 0)
        or (platform.system() == 'Darwin'
        and nb_ipv6 == 0)
```
Small nit

Reply:
I ended up refactoring the assert conditional